sdlib/sys/mman.d

```diff
@@ -11,13 +11,21 @@ enum Prot {
 	Exec = 0x4,
 }
 
+// TODO: confirm platform-specific values
+enum MADV_DONTNEED = 4;
+enum MADV_FREE = 8;
+
 version(OSX) {
 	enum Map {
 		Shared = 0x01,
 		Private = 0x02,
 		Fixed = 0x10,
 		Anon = 0x1000,
 	}
+
+	enum Advise {
+		Purge = MADV_DONTNEED
+	}
```

Comment:
This does not match definitions in mman.h . If there is a notion of purge for the GC, it belongs in the GC.